README.md

```diff
@@ -1,18 +1,20 @@
 # Image Viewer and Annotate
 
-A REDCap external module (EM) that extends the [Image Viewer EM](https://github.com/susom/redcap-em-image-viewer) to annotate uploaded images (using [markerjs2](https://markerjs.com/)) in notes box field types.
+A REDCap external module (EM) that modifies the [Image Viewer EM](https://github.com/susom/redcap-em-image-viewer) to annotate uploaded images (using [markerjs2](https://markerjs.com/)) in notes box field types.
+
+**NOTE**: This EM does not include the [Image Viewer EM's](https://github.com/susom/redcap-em-image-viewer) action tags.
 
 ## Instructions
 
 1. Enable the Image Viewer and Annotate EM through REDCap's control center
```

Comment:
```suggestion
1. Enable the Annotate EM through REDCap's control center
```

Reply:
yeah, we don't need the image viewer enabled anywhere, and we don't clash with it, but we also started from its' code base, so credit where credit is due